src/gui/layout/qgslayoutviewtoolselect.cpp

```diff
@@ -241,14 +243,27 @@ void QgsLayoutViewToolSelect::layoutReleaseEvent( QgsLayoutViewMouseEvent *event
         }
       }
     }
+    else
+    {
+      if ( paperItem )
```

Comment:
can't this be a single else if?

Reply:
Mmm, probably yes, the loop is entered a couple of times, not sure why, but indeed this could be an else if